.travis.yml

```diff
@@ -150,10 +166,16 @@ script:
 
 # run the tests from down here to see if it can work without being at top level
 # only do coverage on the upload machine.
-- if [ "$UPLOAD_DOCS" ]; then
-    testflo -n 1 openmdao --coverage  --coverpkg openmdao --cover-omit \*tests/\*  --cover-omit \*devtools/\* --cover-omit \*test_suite/\* --cover-omit \*docs/\* --cover-omit \*code_review/\*;
-  else
-    testflo -n 1 --timeout=120 openmdao;
+- if [ -z "$SKIP_TESTFLO" ]; then
```

Comment:
could just use `-z "$GUI"` here and no need for SKIP_TESTFLO, right?

Reply:
Having both allows for it to be tested in any other combo in the matrix by putting GUI=1 in that row - for instance in addition to/instead of testing standalone, we might want to test it with the Python 3.6 row because it usually completes faster.